readthedocs/settings/base.py

```diff
@@ -297,6 +297,8 @@ def INSTALLED_APPS(self):  # noqa
             "allauth.socialaccount.providers.gitlab",
             "allauth.socialaccount.providers.bitbucket_oauth2",
             "allauth.mfa",
+            # Others
+            "impersonate",
```

Comment:
Do we want this installed in our base settings, or just on the admin instances? Seems worth putting this before a `if self.ALLOW_ADMIN`, since installing it might have other side effects?

Reply:
Goo point. Yes, I think it's better if we install it only `if self.ALLOW_ADMIN`.

Reply:
It needs to be installed https://github.com/readthedocs/readthedocs-corporate/issues/1346